### ISSUE TITLE
Icons glitching when imported

### ISSUE THREAD

**Issue by sarthaksachdeva13** [ionic-team/ionicons#891]
Hello, 

As a UX Designer, I've been an ardent user of this wonderful framework you provide to the world. Lately, however, there seems to be an issue with the SVGs themselves during import. To elaborate further, whenever I download an icon and try to  import it into Adobe XD or Illustrator, a path or two from the SVG itself seems to be lost in translation and as a result, the resulting import is glitchy and incomplete. I'm not sure if it's the new updated icons, but it's preventing me from creating some super cool designs! I'd appreciate if you could look into it. 
<img width="786" alt="Screen Shot 2020-10-13 at 4 02 57 PM" src="https://user-images.githubusercontent.com/35432319/95913134-ca370100-0d71-11eb-9ee4-732d04445344.png">

Awaiting your response,
Sarthak

**Comment by Miamnamina** [comment]
I have the same issue just a tad more severe ... it's blocking my work as well
<img width="259" alt="Screenshot 2020-10-15 at 11 21 52" src="https://user-images.githubusercontent.com/50818320/96104469-2d8f7300-0ed9-11eb-8a23-732e1244a3d9.png">

**Comment by Jenniferchan3** [comment]
I'm also having the same issue, except when I open this, "business.svg" file in .Ai, it's a completely blank. But on preview, it shows the actual svg. 
![Screen Shot 2020-10-15 at 9 26 27 AM](https://user-images.githubusercontent.com/68825434/96132636-a0442280-0ec8-11eb-8fd8-36921b48f22c.png)

**Comment by juliandegraaf** [comment]
I have the same problem since last week. The layers get all messed up in Sketch. In Photoshop and Illustrator layers are even missing, as already described above in the comments.

![sketch](https://user-images.githubusercontent.com/38209280/96598529-3fb14d00-12ef-11eb-9f93-5dd0e646a4e0.png)

**Comment by chrisgriffith** [comment]
I am having issues with accessibility.svg and accessibility-outline.svg when importing into Adobe XD or Adobe Illustrator.

**Comment by Pixelice35** [comment]
Hi! Same problem here when I open the icons on either Sketch or Illustrator. What seems to be the issue ?

**Comment by viniciuscenci** [comment]
Same here, seems that the svg PATH attribute is malformed for illustrator. 

### Example:
File: phone-portrait-outline.svg

The **path** attribute looks like:

`path="M176 16h24a8 8 0 018 8h0a16 16 0 0016 16h64a16 16 0 0016-16h0a8 8 0 018-8h24"`

When opened in illustrator:

![Screen Shot 2020-11-26 at 4 05 10 PM](https://user-images.githubusercontent.com/2103416/100385508-8f2b1d00-3001-11eb-9026-c11f63333916.png)

Fixed path attribute (works):
`path="M176 16h24a8 8 0 0 1 8 8 h0a16 16 0 0 0 16 16 h64a16 16 0 0 0 16 -16 h0a8 8 0 0 1 8 -8 h24"`

When opened in illustrator:

![Screen Shot 2020-11-26 at 4 04 29 PM](https://user-images.githubusercontent.com/2103416/100385558-b41f9000-3001-11eb-9d5e-44f990888cd0.png)


In this case, the **arc** operator 'a' has to be followed by 7 parameters...
Take the below portion:
`...a8 8 0 018 8h...`

Should be:
`...a8 8 0 0 1 8 8 h...`

I think the svgs need to be re-exported...

**Comment by KyleTryon** [comment]
Adding on here. I am attempting to make a variant of the water icon with the "off" line through it, downloaded the pack but barely any will open properly in Illustrator.

**Comment by webjac** [comment]
Same thing here. Most icons break when importing into Sketch. I'm guessing there's some information missing or mishandled in the files that Sketch doesn't recognize well.

**Comment by marijnvb** [comment]
Same issue here. Is there a work around or solution for this?
![EoUw9raWEAER54t](https://user-images.githubusercontent.com/713095/101623843-4ae44780-3a19-11eb-8729-58977fae9be5.jpg)

**Comment by MathiasDenti** [comment]
Same problem for me in both Illustrator and Photshop...

**Comment by brolnickij** [comment]
same problem :(

**Comment by GeorgeHulpoi** [comment]
+1

**Comment by jrj2211** [comment]
same problem

**Comment by rtvt** [comment]
any news on this? By the way 5.0.0 pack don't have this problem with outline icons. Now I can't use any outline icon in any Adobe products.

**Comment by jrj2211** [comment]
I did try converting these with several online sites to convert SVG to EPS to see if that would at least be a work around - none of them opened in Adobe products. Uploading the SVG's to online validators says its correct and seems like browsers can always open them. I also tried several "SVG Editors" apps off the Microsoft Store and the icons opened fine in those. I'm thinking its an issue with Adobe and not ionicons. Is there a new format that these are being exported in that Illustrator & PS might not _yet_ support?

**Comment by KyleTryon** [comment]
CC @adamdbradley sorry for the ping, saw you on the first commit for some of the affected icons. Could we ask for an update/comment on this?

**Comment by marijnvb** [comment]
@jrj2211 I agree on your comment! I tried the same, convert online to EPS, import in Sketch App. That works. I think we need to knock on the door at Adobe and Sketch for our problem.

**Comment by viniciuscenci** [comment]
Possible workaround:

Open with Chrome and Print as PDF... then it worked just fine when opened in illustrator..

**Comment by marijnvb** [comment]
I contacted Sketch support for this issue. This is their response:

> Hi Marijn,
> 
> We’re always working to improve our import and export support for SVG files. Some of these files use specific properties that we do not yet support though.
> As you have noted, this issue occurs in other applications as well, such as Adobe Illustrator, so it is not unique to Sketch. Ionic themselves may be able to provide you with more information as to how they are creating / exporting these files.
> However, as a workaround, I did notice the SVG looked OK in Inkscape (free), so if you export it from there as an EPS, and then import the EPS instead into Sketch, this should now work OK. 
> 
> I hope this helps, and if there is anything else we can help you with, be sure to let us know.

**Comment by christian-bromann** [comment]
@marijnvb thanks for providing this workaround. I will go ahead and close the issue.
